todo_app/data/view_model.py

```diff
@@ -0,0 +1,23 @@
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+ 
+    @property
+    def items(self):
+        return self._items
+
+    @property
+    def done_items(self):
+        done_item_list = []
+        for item in self._items:
+            if item.status == 'Done':
+                done_item_list.append(item)
+        return done_item_list
```

Comment:
I wouldn't necessarily recommend it, but I do think worth comparing how we might do this more concisely (but arguably less readably) with list comprehensions:
```python
        return [item for item in self._items if item.status == "Done]
```